starling/core/RenderSupport.hx

```diff
@@ -748,4 +759,14 @@ class RenderSupport
     /** Indicates the number of stage3D draw calls. */
     public var drawCount(get, never):Int;
     private function get_drawCount():Int { return mDrawCount; }
+    
+    public var batcher(get, never):BatchRenderer;
+    inline function get_batcher() return @:privateAccess Starling.current.context.__renderSession.batcher;
+    
+    public function updateBatchersProjectionMatrix(): Void {
+        for (i in 0...16) {
+                sBatcherProjectionMatrix[i] = projectionMatrix3D.rawData[i];
+        }
+        batcher.projectionMatrix = sBatcherProjectionMatrix;
+    }
```

Comment:
Would it make sense to always keep `batcher.projectionMatrix` in sync with `mProjectionMatrix3D` by always updating both together?

Reply:
Sure, it was like this at the beginning, but we need small price to pay. Namly, `mProjectionMatrix3D` can be updated before the context is created. Therefore to having it in sync we need to check if the batcher exists and only then update the matrix.